pkg/manager/setconfig.go

```diff
@@ -121,6 +123,64 @@ func (m *Manager) SetNetworkConfig(deviceName string, version string,
 	return changeID, configName, nil
 }
 
+// SetNewNetworkConfig creates and stores a new netork config for the given updates and deletes and targets
+func (m *Manager) SetNewNetworkConfig(targetUpdates map[string]devicechangetypes.TypedValueMap,
+	targetRemoves map[string][]string, version string, deviceType string, netcfgchangename string) {
+	//TODO evaluate need of user and add it back if need be.
+	//TODO start watch and build update Result
+	allDeviceChanges, errChanges := m.computeNewNetworkConfig(targetUpdates, targetRemoves, version, deviceType, netcfgchangename)
+	if errChanges != nil {
+		log.Error("Can't compute new network configs", errChanges)
+	}
+	newNetworkConfig, errNetConfig := networktypes.NewNetworkConfiguration(netcfgchangename, allDeviceChanges)
```

Comment:
maybe `newNetworkChange` instead of `newNetworkConfig` - also `errNetChange`

Reply:
done